src/ucp/core/ucp_worker.h

```diff
@@ -59,9 +59,14 @@
  * UCP worker flags
  */
 enum {
-    UCP_WORKER_FLAG_EXTERNAL_EVENT_FD = UCS_BIT(0), /**< worker event fd is external */
-    UCP_WORKER_FLAG_EDGE_TRIGGERED    = UCS_BIT(1), /**< events are edge-triggered */
-    UCP_WORKER_FLAG_MT                = UCS_BIT(2)  /**< MT locking is required */
+    /** MT locking is required */
```

Comment:
are values moved to allow combining with new `params->flags`?
Can you please add a comment and some check they are not mixed?